app/src/main/java/org/fossasia/openevent/general/event/EventService.kt

```diff
@@ -25,4 +27,16 @@ class EventService(private val eventApi: EventApi, private val eventDao: EventDa
         return eventDao.getEvent(id)
     }
 
+    fun getTickets(id: Long): Single<List<Ticket>> {
```

Comment:
Flowable should be returned

Reply:
I don't know why but the process of loading tickets was looping when i was using Flowable, I changed it to Single and the looping didn't occur

Reply:
Looping?

Reply:
Yeah Like the call for tickets is being done inspite of tickets received. Like the doFinally() method of TicketViewModel is never being called.

Reply:
Change this to flowable and I'll see